[MIG] partner_identification: Migration to 15.0

jdidderen-noviat opened the pull request:
Since the PR #1270 seems to be (almost) abandoned, I backported the work done by @marielejeune for v16 in the PR #1397

lmignon committed:
[IMP] Rename partner_identifiers -> partner_identification

lmignon committed:
partner_identification: several fix/imp

* [IMP] improve module description
* [IMP] Remove useless comments
* [FIX] Complete incomplete sentence
* [IMP] Replace field 'state' by 'status' in res_partner_id_number
* [IMP] Add new field 'Place of Issuance'
* [FIX] Readme formatting
* [IMP] status is now a selection field
* [IMP] use method to provide the default value for validation_code
* [IMP] Add help texts
* [FIX] Add missing constrains on category_id in res_partner.id_number
        The number must be validated also when we change the category

oca-transbot committed:
OCA Transbot updated translations from Transifex

lmignon committed:
[FIX] partner_identification: prevent integrity error on delete ID

jjscarafia committed:
[ADD] partner_identification: sales_team dep

jjscarafia committed:
[FIX] partner_identification: pylint

lasley committed:
[IMP] partner_identification: Add context override (#373)

Allow for context override of validations using ``id_no_validate``

lasley committed:
[10.0][IMP] partner_identification: Add field computation and inverses (#419)

* [IMP] partner_identification: Add field computation and inverses
* Add methods to allow for computation and inverse of an ID field of a specific category type

* [IMP] partner_identification: Add search option

lasley committed:
[FIX] partner_identification: Infinite loop in search (#436)

simahawk committed:
[11.0][MIG] partner_identification

simahawk committed:
partner_identification: fix/imp tests on res.partner

oca-travis committed:
[UPD] Update partner_identification.pot

tarteo committed:
[MIG] partner_identification: Migration to 12.0

[FIX] Tests

[FIX] data sequence

Fix openerp to odoo in comment

[REM] at_install, post_install

[ADD] Model descriptions

[REM] Extra spaces

OCA-git-bot committed:
[UPD] README.rst

oca-travis committed:
[UPD] Update partner_identification.pot

primes2h committed:
Translated using Weblate (Italian)

Currently translated at 97.9% (46 of 47 strings)

Translation: partner-contact-12.0/partner-contact-12.0-partner_identification
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-12-0/partner-contact-12-0-partner_identification/it/

oca-transbot committed:
Update translation files

Updated by Update PO files to match POT (msgmerge) hook in Weblate.

oca-travis committed:
[UPD] Update partner_identification.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: partner-contact-12.0/partner-contact-12.0-partner_identification
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-12-0/partner-contact-12-0-partner_identification/

mvrodriguez committed:
Translated using Weblate (Spanish)

Currently translated at 97.9% (46 of 47 strings)

Translation: partner-contact-12.0/partner-contact-12.0-partner_identification
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-12-0/partner-contact-12-0-partner_identification/es/

OCA-git-bot committed:
[ADD] icon.png

OCA-git-bot committed:
[UPD] README.rst

Commit:
[REF] partner_identification: Black python code

Commit:
[MIG] partner_identification: Migration to 13.0

ntsirintanis committed:
[FIX] make _compute_identification() to assign a value no matter what

oca-travis committed:
[UPD] Update partner_identification.pot

OCA-git-bot committed:
[UPD] README.rst

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: partner-contact-13.0/partner-contact-13.0-partner_identification
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-13-0/partner-contact-13-0-partner_identification/

OCA-git-bot committed:
pre-commit update

dong-z committed:
Translated using Weblate (Chinese (Simplified))

Currently translated at 100.0% (47 of 47 strings)

Translation: partner-contact-13.0/partner-contact-13.0-partner_identification
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-13-0/partner-contact-13-0-partner_identification/zh_CN/

OCA-git-bot committed:
[UPD] README.rst

TDu committed:
[FIX][13.0] partner_identification id number check

OCA-git-bot committed:
partner_identification 13.0.1.0.1

ajaniszewska-dev committed:
[FIX][13.0] partner_identification: Add menu for Partner ID Numbers

oca-travis committed:
[UPD] Update partner_identification.pot

OCA-git-bot committed:
partner_identification 13.0.1.1.0

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: partner-contact-13.0/partner-contact-13.0-partner_identification
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-13-0/partner-contact-13-0-partner_identification/

ajaniszewska-dev committed:
[13.0][IMP] Add missing field to ID Numbers tree view

OCA-git-bot committed:
partner_identification 13.0.1.2.0

ibuioli committed:
Translated using Weblate (Spanish (Argentina))

Currently translated at 100.0% (47 of 47 strings)

Translation: partner-contact-13.0/partner-contact-13.0-partner_identification
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-13-0/partner-contact-13-0-partner_identification/es_AR/

bosd committed:
Translated using Weblate (Dutch)

Currently translated at 21.3% (10 of 47 strings)

Translation: partner-contact-13.0/partner-contact-13.0-partner_identification
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-13-0/partner-contact-13-0-partner_identification/nl/

phuctranfxvn committed:
[IMP] partner_identification: black, isort, prettier

phuctranfxvn committed:
[MIG] partner_identification: Migration to 14.0

phuctranfxvn committed:
[IMP] partner_identification: Add python validation, improve view and fix default partner

phuctranfxvn committed:
[FIX] partner_identification: Do not validate number if validation code is not set

Chanakya-OSI committed:
[14.0][IMP] Added the field 'color' to partner_id_category.

educasilva committed:
Translated using Weblate (Portuguese (Portugal))

Currently translated at 100.0% (47 of 47 strings)

Translation: partner-contact-14.0/partner-contact-14.0-partner_identification
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-14-0/partner-contact-14-0-partner_identification/pt_PT/

OCA-git-bot committed:
partner_identification 14.0.1.3.0

pedrocs-exo committed:
Translated using Weblate (Portuguese (Portugal))

Currently translated at 97.9% (48 of 49 strings)

Translation: partner-contact-14.0/partner-contact-14.0-partner_identification
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-14-0/partner-contact-14-0-partner_identification/pt_PT/

pedrocs-exo committed:
Translated using Weblate (Portuguese)

Currently translated at 97.9% (48 of 49 strings)

Translation: partner-contact-14.0/partner-contact-14.0-partner_identification
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-14-0/partner-contact-14-0-partner_identification/pt/

rbellanova committed:
Translated using Weblate (Italian)

Currently translated at 91.8% (45 of 49 strings)

Translation: partner-contact-14.0/partner-contact-14.0-partner_identification
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-14-0/partner-contact-14-0-partner_identification/it/

rbellanova committed:
Translated using Weblate (Italian)

Currently translated at 91.8% (45 of 49 strings)

Translation: partner-contact-14.0/partner-contact-14.0-partner_identification
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-14-0/partner-contact-14-0-partner_identification/it/

marielejeune committed:
[MIG] partner_identification: Migration to 16.0

Commit:
[UPD] Update partner_identification.pot

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
partner_identification 16.0.1.0.1

rousseldenis commented:
/ocabot migration partner_identification

rousseldenis commented:
@jdidderen-noviat Thanks for this.

Could you follow migration guide ? https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-15.0

Especially the Full process part (inversing branches in git format-patch command) ? 

You must keep commit authorship from @marielejeune ones

njeudy reviewed (approved):
code and test

njeudy commented:
I think @marielejeune do the work in V16 and here we patch from v14 .. so .. he add marielejeune as contributor .. that ok for me I think :)

badbole committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: partner-contact-16.0/partner-contact-16.0-partner_identification
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-16-0/partner-contact-16-0-partner_identification/

Translated using Weblate (Italian)

Currently translated at 87.7% (43 of 49 strings)

Translation: partner-contact-16.0/partner-contact-16.0-partner_identification
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-16-0/partner-contact-16-0-partner_identification/it/

Translated using Weblate (Italian)

Currently translated at 87.7% (43 of 49 strings)

Translation: partner-contact-16.0/partner-contact-16.0-partner_identification
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-16-0/partner-contact-16-0-partner_identification/it/

Translated using Weblate (Croatian)

Currently translated at 30.6% (15 of 49 strings)

Translation: partner-contact-16.0/partner-contact-16.0-partner_identification
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-16-0/partner-contact-16-0-partner_identification/hr/

Translated using Weblate (Croatian)

Currently translated at 51.0% (25 of 49 strings)

Translation: partner-contact-16.0/partner-contact-16.0-partner_identification
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-16-0/partner-contact-16-0-partner_identification/hr/

jdidderen committed:
[IMP] partner_identification: black, isort, prettier

ghost commented:
@rousseldenis Sorry for the delay. I was a bit silly when I did this PR. I did the format patch from 14.0 to 15.0 and then cherry pick the changes from @marielejeune instead of format patching from 16.0 to 15.0

@njeudy It was exactly that. 

Thanks for the reviews :)

NL66278 commented:
@jdidderen-noviat Please squash some of the translation commits (for instance the consecutive ones for Croatian or Italian).

ghost commented:
@NL66278 Done

MiquelRForgeFlow commented on partner_identification/__manifest__.py:
```diff
@@ -0,0 +1,30 @@
+{
+    "name": "Partner Identification Numbers",
+    "category": "Customer Relationship Management",
+    "version": "15.0.1.0.1",
```
```suggestion
    "version": "15.0.1.0.0",
```

jdidderen committed:
[MIG] partner_identification: Migration to 15.0

dreispt commented:
/ocabot merge nobump
